src/components/maps/StationPopup/_x__tests__/smoke.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
why is this file in an oddly named directory?

Reply:
To take it out of test detection without deleting it. Kinda hanging on till later.